### DISCUSSION TITLE
In panoptic segmentation using Detectron2, how can I output the instance ID data in the format in which pixel? (Not in image format)

### DISCUSSION THREAD

**Discussion by shutofuji** [facebookresearch/detectron2#3711] [Q&A]
I was able to run demo.py and output panoptic segmentation. However, the output is an image only. Is there a way to output as a code such as a JSON file instead of an image?
The data you want is the instance ID and its pixels.

**Comment by ppwwyyxx** [comment]
You can write code to convert the output into formats you like.
